src/Microsoft.DotNet.ProjectModel/PackOptions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class PackOptions
```

Comment:
When I wrote the spec I moved metadata that was package-specific to `packOptions` but was never really happy with it. I'd expected people to look at the spec and give feedback so we could refine it before it would be worked on. It's probably cleaner to leave all the metadata in the root, but it still feels a little "grab bag of crap" to me. :confounded: